.travis.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     - stage: analyze_and_format
       name: "Analyze lib/"
       os: linux
-      script: dartanalyzer --enable-experiment=non-nullable --fatal-warnings --fatal-infos lib/
+      script: dartanalyzer --fatal-warnings --fatal-infos lib/
```

Comment:
We should be able to merge these two jobs now actually (without the flag for either). There was an issue previously where the allow list didn't apply to non-package: uris, but that has been fixed.